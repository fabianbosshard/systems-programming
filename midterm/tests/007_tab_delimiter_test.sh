exec "$1" 'format=@b - @a' delim='\t' <<EOF
Pineapple\t12\t7
Mango\t10\t5
EOF
