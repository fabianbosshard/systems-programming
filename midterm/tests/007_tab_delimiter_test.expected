12 - Pineapple
10 - Mango
